packages/docusaurus-plugin-content-docs/src/__tests__/frontMatter.test.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import {escapeRegexp} from '@docusaurus/utils';
-import {validateDocFrontMatter} from '../frontMatter';
```

Comment:
revert

BTW tests and the tested code should remain colocated. It doesn't make sense to add tests for `@docusaurus/utils-validation'` outside of `@docusaurus/utils-validation';` like it now became the case here. 

pkg1 tests should test pkg1 code, while now we have pkg2 tests testing pkg1 code.